src/components/validations/validation-icon.component.js

```diff
@@ -31,7 +32,13 @@ const ValidationIcon = ({
     <InputPresentationContext.Consumer>
       {
         context => (
-          <ValidationIconStyle type={ type }>
+          <ValidationIconStyle
+            id={ iconId }
+            type='button'
+            data-validation-type={ type }
+            validationType={ type }
+            aria-label='additional validation information'
```

Comment:
This should not be hard coded. The aria-label should be the contents of the tooltip, it also needs the role "tooltip".
To see a working a11y tooltip example check out https://van11y.net/downloads/simple-tooltip/demo/index.html using VoiceOver on your Mac.

Reply:
In fact, aria-describedby should do the job, but I can't remember if the tooltip has to be visible.
docs: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute
example:
http://www.oaa-accessibility.org/examplep/tooltip1/

Reply:
@jamime Updated in commit 93fd9b571

Reply:
@jamime I believe I have resolved the issue with the input, where the input should now be linked with the label and not inherit the tooltip aria-label as well.
Commit 4d91dfbb4.